ERCS/erc-7570.md

```diff
@@ -0,0 +1,98 @@
+---
+eip: 7570
```

Comment:
```suggestion
eip: 7582
```
Assigning next sequential EIP/ERC/RIP number.  ERC7570 has already been assigned.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.